packages/docs/docs/contracts-api/epoch-key-lite-proof-verifier.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Epoch Key Lite Proof Verifier Contract
+---
+
+This smart contract is dedicated to verifying epoch key lite proofs
+
+## decodeEpochKeyLiteSignals
+
+Decode the public signals from an [epoch key lite proof](../circuits-api/circuits#epoch-key-lite-proof) info named variables.
+
+```sol
+function decodeEpochKeyLiteSignals(uint256[] memory publicSignals)
+    public
+    pure
+    returns (EpochKeySignals memory)
+```
+
+:::tip
+The `stateTreeRoot` variable in this struct is unused for epoch key lite proofs.
+:::
+
+```sol
+struct EpochKeySignals {
+    bool revealNonce;
+    uint8 nonce;
+    uint48 epoch;
+    uint160 attesterId;
+    uint256 stateTreeRoot;
+    uint256 epochKey;
+    uint256 data;
+}
+```
+
+## verifyAndCheck
+
+Verify an [epoch key lite proof](../circuits-api/circuits#epoch-key-lite-proof) and validate the public signals against the onchain state. This function will revert if any inputs are invalid.
+
+:::caution
```

Comment:
Here we should also add a `:::danger:::` note about verifying the `attesterId`. We should recommend using `verifyAndCheckCaller` or manually checking the `attesterId`. Same with other proof contracts.

Reply:
`verifyAndCheckCaller` recommendation added